kroxylicious-api/src/main/java/io/kroxylicious/proxy/filter/FilterFactory.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.kroxylicious.proxy.filter;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+/**
+ * FilterFactory is a pluggable source of Kroxylicious filter implementations.
+ * @param <F> the {@link Filter} type
+ * @param <C> the configuration type for the Filter (use {@link Void} if the Filter is not configurable)
```

Comment:
```suggestion
 * @param <C> the type of configuration {@code Filter}. Use {@link Void} if the {@code Filter} is not configurable.
```

Reply:
I added this manually with some tweaks to grammar